src/gui/torrentcontentmodelitem.cpp

```diff
@@ -112,42 +112,39 @@ QString TorrentContentModelItem::displayData(const int column) const
     case COL_NAME:
         return m_name;
     case COL_PRIO:
-    {
-            switch (m_priority)
-            {
-            case BitTorrent::DownloadPriority::Mixed:
-                return tr("Mixed", "Mixed (priorities");
-            case BitTorrent::DownloadPriority::Ignored:
-                return tr("Not downloaded");
-            case BitTorrent::DownloadPriority::High:
-                return tr("High", "High (priority)");
-            case BitTorrent::DownloadPriority::Maximum:
-                return tr("Maximum", "Maximum (priority)");
-            default:
-                return tr("Normal", "Normal (priority)");
-            }
+        switch (m_priority)
+        {
+        case BitTorrent::DownloadPriority::Mixed:
+            return tr("Mixed", "Mixed (priorities");
+        case BitTorrent::DownloadPriority::Ignored:
+            return tr("Not downloaded");
+        case BitTorrent::DownloadPriority::High:
+            return tr("High", "High (priority)");
+        case BitTorrent::DownloadPriority::Maximum:
+            return tr("Maximum", "Maximum (priority)");
+        default:
+            return tr("Normal", "Normal (priority)");
         }
     case COL_PROGRESS:
-    {
-            const qreal progress = m_progress * 100;
-            return (static_cast<int>(progress) == 100)
+        {
```

Comment:
So braces isn't neede anymore in this branch.

Reply:
Removed.